### PR TITLE
docs: adds notification log docs and updates alerting docs in general

### PR DESCRIPTION
**Pull request by tnolet**
## Affected Components
* [ ] Content & Marketing
* [ ] Test
* [x] Docs
* [ ] Learn
* [ ] Other

## Pre-Requisites
* [ ] Code is linted (`npm run lint`)

## Notes for the Reviewer
- adds section on notification logs
- updates all the relevant screenshots in alerting section. They were very very old.

### CODE REVIEW

**Comment by stefanjudis** [comment]
@tnolet Looking at the text changes now but do you think we can do something about the images? They're averaging on 450KB?

File: site/content/docs/alerting/_index.md [hunk 1]
```diff
@@ -19,23 +19,28 @@ for all options.
 
 ## Alert settings
 
-The alert settings tab gives you a lot of options to tailor when, how and how often you want to be alerted when
-a check fails. This is also sometimes referred to as **threshold alerting**. For example:
+The [alert settings screen](https://app.checklyhq.com/alerts/settings) gives you a lot of options to tailor when, how and how
```
Comment (stefanjudis) [inline site/content/docs/alerting/_index.md:22]:
```suggestion
The [alert settings screen](https://app.checklyhq.com/alerts/settings) gives you the options to tailor when, how and how 
```

File: site/content/docs/alerting/_index.md [hunk 2]
```diff
@@ -19,23 +19,28 @@ for all options.
...
-a check fails. This is also sometimes referred to as **threshold alerting**. For example:
+The [alert settings screen](https://app.checklyhq.com/alerts/settings) gives you a lot of options to tailor when, how and how 
+often you want to be alerted when a check fails. This is also sometimes referred to as **threshold alerting**. For example:
 
 - Get an alert on the second or third failure.
 - Get an alert after 5 minutes of failures.
 - Get one or more reminders after a failure is triggered.
 
-Configuration is really simple. Just remember that you can configure your alert settings at two levels:
+Configuration is really simple. Just remember that you can configure your alert settings at three levels:
```
Comment (stefanjudis) [inline site/content/docs/alerting/_index.md:29]:
```suggestion
Your alert notifications can be configured at three levels:
```

File: site/content/docs/alerting/_index.md [hunk 3]
```diff
@@ -19,23 +19,28 @@ for all options.
...
 
-- **Account** level: This is the default level and applies to all of your check unless you override these settings at the check level.
+1. **Account** level: This is the default level and applies to all of your check unless you override these settings at the check level.
```
Comment (stefanjudis) [inline site/content/docs/alerting/_index.md:31]: Or group, or?

File: site/content/docs/alerting/alert-notification-log.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+---
+title: Alert notification log
+weight: 36
+menu:
+  docs:
+    parent: "Alerting"
+---
+ 
+Checkly tries to deliver all your failures, degradation and recovery notifications as reliably as possible, but sometimes
```
Comment (stefanjudis) [inline site/content/docs/alerting/alert-notification-log.md:9]:
```suggestion
Checkly tries to deliver all your failure, degradation and recovery notifications as reliably as possible, but sometimes
```

File: site/content/docs/alerting/alert-notification-log.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+life happens. A token expires, a URL has a typo or some other unforeseen issue causes a notification to get lost.
+
+To give you insight and troubleshoot any delivery issues, you can browse and filter the
```
Comment (stefanjudis) [inline site/content/docs/alerting/alert-notification-log.md:12]:
```suggestion
To troubleshoot any delivery issues, browse and filter the 
```

File: site/content/docs/alerting/alert-notification-log.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+Here's an example:
+
+![alert notification log](/docs/images/alerting/alert-notification-log.png)
+
+What do we see here?
+
+1. SMS and Email were correctly delivered.
+2. The OpsGenie notification failed though. The notification result shows we got a `422` status code back and the error 
+message states our API key was invalid.
+3. We also see the configuration we used to instrument our call to the OpsGenie API.
```
Comment (stefanjudis) [inline site/content/docs/alerting/alert-notification-log.md:25]:
```suggestion
3. We also see the configuration used to instrument our call to the OpsGenie API.
```

**Review by stefanjudis — DISMISSED** [review]
I just left some rewordings here and there to spare some words. :) 

Feel free to move forward or accept the changes. :🤷‍♂️

**Comment by tnolet** [comment]
> @tnolet Looking at the text changes now but do you think we can do something about the images? They're averaging on 450KB?

@stefanjudis I just pushed a version with some shrunk down images

**Review by stefanjudis — APPROVED** [review]
:shipit:

**Pull request merged**